lib/blogs_api/post/create.ex

```diff
@@ -1,3 +1,15 @@
 defmodule BlogsApi.Post.Create do
+  @moduledoc """
+  Módulo que busca, valida e insere as
+  informações necessárias para criação do post com base no id do usuário
```

Comment:
```suggestion
  informações necessárias para criação do POST com base no `id` do usuário
```

Algo meio bobo, mas acho legal para legibilidade da documentação, assim "dizendo" que o POST é uma request POST, e o `id` seja algo de código, quando é dentro das aspas ``

Reply:
Bacana, Eric. Não sabia disso. Obrigada por informar. Irei corrigir!!! :D